cmd/virt-launcher/virt-launcher.go

```diff
@@ -219,6 +219,11 @@ func initializeDirs(ephemeralDiskDir string,
 	if err != nil {
 		panic(err)
 	}
+
+	err = virtlauncher.InitializeDisksDirectories(downwardmetrics.DownwardMetricsChannelDir)
```

Comment:
supernit - inline all of these but that's something for a follow up.

Reply:
Oks, I will address it in a separate PR.